api/app/models/position.rb

```diff
@@ -13,11 +13,10 @@ class Position < ApplicationRecord
   has_many :assignments
   has_many :position_preferences
   has_many :applications, through: :position_preferences
+  has_one :position_data_for_ad
+  has_one :position_data_for_matching
 
 
-  validates_presence_of :course_code, :openings, :session
```

Comment:
Is there a reason we're deleting these rails validations? I don't think we have the database validations for these either.

Reply:
I deleted these validation because the columns used for the validation no longer exist. The new schema plan got rid of them, so I wasn't if the exact same constraints should apply to which attribute.